src/main/java/org/reactome/orthoinference/SkipInstanceChecker.java

```diff
@@ -67,6 +57,13 @@ public static boolean checkIfInstanceShouldBeSkipped(GKInstance reactionInst) th
 			logger.info(reactionInst + " is in skipList -- skipping");
 			return true;
 		}
+
+		// If the only TopLevelPathway of a Reaction is 'Disease', then it is skipped.
+		// Otherwise, it is inferred, making sure in cases where a Reaction is also a part of 'Disease' that that Pathway is not inferred.
+		if (org.reactome.orthoinference.InstanceUtilities.onlyInDiseasePathway(reactionInst)) {
```

Comment:
Why is `InstanceUtilities` fully qualified with `org.reactome.orthoinference`?

Reply:
There is a coincident class name (InstanceUtilities) in orthoinference and in the CuratorTool repo. Both are called in this class

Reply:
If the `onlyInDiseasePathway` is statically imported, it should remove the need for the qualification, but I'm not 100% sure.

Reply:
I changed it so that one of the methods is statically imported

Reply:
I would have been happy with just the explanation - I don't like static imports. ;)